picoquicfirst/picoquicdemo.c

```diff
@@ -679,6 +689,12 @@ int quic_client(const char* ip_address_text, int server_port,
 
     /* Wait for packets */
     if (ret == 0) {
+        struct sockaddr_in alt_addr;
+        memset(&alt_addr, 0, sizeof(alt_addr));
+        alt_addr.sin_family = AF_INET;
+        alt_addr.sin_addr.s_addr = inet_addr(config->multipath_alternative_ip);
+        memcpy(&loop_cb.client_alt_address, &alt_addr, sizeof(alt_addr));
+
```

Comment:
This code uses "inet_addr(), which is not portable. We get the following warning treated as error in Windows builds:
```
c:\projects\picoquic\picoquicfirst\picoquicdemo.c(695): error C2220: warning treated as error - no 'object' file generated [C:\projects\picoquic\picoquicfirst\picoquicfirst.vcxproj]
    21>c:\projects\picoquic\picoquicfirst\picoquicdemo.c(695): warning C4996: 'inet_addr': Use inet_pton() or InetPton() instead or define _WINSOCK_DEPRECATED_NO_WARNINGS to disable deprecated API warnings [C:\projects\picoquic\picoquicfirst\picoquicfirst.vcxproj]
         c:\program files (x86)\windows kits\10\include\10.0.18362.0\um\winsock2.h(1831): note: see declaration of 'inet_addr'
```
Please use the utility function from the library:
```
int picoquic_store_text_addr(struct sockaddr_storage* stored_addr, const char* ip_address_text, uint16_t port)
```
It is portable, not obsoleted, and will also automatically deal with IPv4/IPv6 issues. Set port to 0 if you only care about the IP address.